pkg/util/sys/device.go

```diff
@@ -91,6 +102,59 @@ func ListDevices(executor exec.Executor) ([]string, error) {
 	return strings.Split(devices, "\n"), nil
 }
 
+func GetCephVolumeInventory(executor exec.Executor) (*[]CephVolumeDeviceData, error) {
```

Comment:
I'd suggest moving these c-v methods up to the `cmd/daemon/discover` package. The `sys` package is more for things closely tied to the system. If we're executing ceph-volume process, it doesn't feel like a `sys` operation.

Reply:
Probably you mean `pkg/daemon/discover`. I have moved this function to this new place. Agree with you about that this is not a sys operation.